python/py-catalogue/Portfile

```diff
@@ -38,7 +38,12 @@ if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
-    depends_run-append \
+    if {${python.version} < "38"} {
+        depends_build-append \
+                    port:py${python.version}-importlib-metadata
+    }
+
+    depends_test-append \
                     port:py${python.version}-pytest
```

Comment:
you're adding a test dependency, but no test-phase - that does not make sense. Please add a ```test``` phase so that the tests can be run.

Reply:
There’s some test script, but I don’t know how to invoke it from MacPorts. See https://github.com/explosion/catalogue/blob/master/test_catalogue.py

Please post code if you know the approach.

Reply:
what have you tried? By now you’ve encountered several varieties (i.e., using py.test-${python.branch} or ${python.bin}) - I am sure that one of those approaches will work.

Reply:
I haven’t enabled test.run because afaik there’s no magic that let’s it find the test script linked above. Is there? If so, I’ll just enable it and hope it works.

Reply:
Done thanks.